lib/perl/Genome/Model/Build/Command/Export.pm

```diff
@@ -44,7 +63,9 @@ sub execute {
     $resolve_symlinks = sub {
         my $path = $File::Find::name;
         if (-l $path) {
+            #print STDERR "path: $path\n";
```

Comment:
I'd either remove these commented-out `print`s or convert them to `debug_message`s if they might be useful more generally.